src/components/tiltak/HvaErDette.tsx

```diff
@@ -0,0 +1,28 @@
+import { BodyLong, Detail, Heading } from '@navikt/ds-react'
+import { formatDateStrWithMonthName } from '../../utils/utils.ts'
+import { Vedtaksinformasjon } from '../../api/data/pamelding.ts'
+
+interface Props {
+  vedtaksinformasjon: Vedtaksinformasjon
+}
+
+export const HvaErDette = ({ vedtaksinformasjon }: Props) => {
+  const harDeltakerGodkjent = vedtaksinformasjon.fattet
+
+  return (
+    <div>
+      <Heading level="2" size="medium">
+        Hva er dette?
+      </Heading>
+      <BodyLong size="small">
+        Dette er et vedtak etter arbeidsmarkedsloven § 12 og forskrift om arbeidsmarkedstiltak
+        kapittel 4.
+      </BodyLong>
+      <Detail className="mt-2">
+        {harDeltakerGodkjent
+          ? `Utkast delt av: ${vedtaksinformasjon.sistEndretAv}. Du godkjente ${formatDateStrWithMonthName(vedtaksinformasjon.fattet)}.`
+          : `Meldt på: ${formatDateStrWithMonthName(vedtaksinformasjon.sistEndret)} av ${vedtaksinformasjon.sistEndretAv}.`}
```

Comment:
`sistEndret` og `sistEndretAv` bør stemme overens med hvem og når vedtaket ble fattet, men for å vær helt trygg var det tenkt å bruke `fattet` og `fattetAvNavVeileder` her.

Reply:
Var litt i stuss, her, er så mange å velge mellom, vi har også sist endret på deltaker-nivå, der har vi også nav-enhet? Hvis fattet og  fattetAvNavVeileder kan være navident, er det greit? skal vi ikke vise navnet?